app/services/personal_key_generator.rb

```diff
@@ -37,7 +37,7 @@ def normalized_code(plaintext_code)
     normed = plaintext_code.gsub(/\W/, '')
     split_length = normed.length / length
     decoded = Base32::Crockford.decode(normed)
-    Base32::Crockford.encode(decoded, length: 16, split: split_length).tr('-', ' ')
```

Comment:
@pkarman I took your guidance and it worked out mostly as expected.  The one gotcha is that in a couple scenarios we send the personal key off untouched to decrypt pii, like [this](https://github.com/18F/identity-idp/blob/master/app/controllers/two_factor_authentication/personal_key_verification_controller.rb#L42), so by leaving it with a `-` delimiter I was getting a ciphertext error.  I added a `tr` in the model [here](https://github.com/18F/identity-idp/pull/1328/files#diff-6ef5e2d67a2bf4ad80d150039dccd3f2R36), but not sure if this is the best place for it, or the right overall approach.